src/scicookie/{{cookiecutter.project_slug}}/build-system/flit-pyproject.toml

```diff
@@ -120,6 +120,11 @@ dependencies = [
     "jupyter-book >= 0.15.1",
     "myst-parser >= 0.18.1",
 {% endif %}
+{%- if cookiecutter.automation_tools == "make" %}
+    "make >=0.1.6",
```

Comment:
`make` (the correct one) is not available on pypi .. this one in pypi is a different project
for  `make`, just add it to the conda env file 
you just need to check here if the automation tool is makim, and add it if it is selected.

the same applies to all the other build system files